src/ROOT-Sim.h

```diff
@@ -10,23 +10,8 @@
  * a simulation model. All function prototypes exposed to the application
  * developer are exposed and defined here.
  *
- * @copyright
```

Comment:
The @copyright tag allows to generate the Doxygen documentation consistently. Should we keep it as is and have SPDX in addition? Should we keep it but have only SPDX? Should we drop this Doxygen tag?

Reply:
This problem is addressed by a proper doxygen filter script, which you can find at _scripts/doxy_filter.py_
SPDX tags are transformed into equivalent **@copyright** Doxygen tags.